### DISCUSSION TITLE
Looking for a .x86 bits windows version of Qfields.

### DISCUSSION THREAD

**Discussion by Strunge29** [opengisch/QField#3232] [Ideas]
can anyone tell me where can get a .x86 version of qfield for windows. I've been searching so hard but still nothing. Why will anyone make a software opensource and then restrict usage to a specific processor. Doesn't seem to make sense to me. Am I missing something?

**Comment by m-kuhn** [comment]
This is being discussed in https://github.com/opengisch/QField/issues/3231

We are a friendly community, making software free and open-source available for everyone. What is currently available is the result of efforts of people who have invested time and/or money into it. You should be thankful.
We are welcoming everyone to contribute and help make things better.
